app/guests+api.ts

```diff
@@ -36,17 +36,13 @@ export async function POST(request: Request): Promise<Response> {
     );
   }
 
-  console.log('first name', body.firstName);
-  console.log('last name', body.lastName);
-
-  const guest: Guest = {
-    id: guests.length + 1,
-    firstName: body.firstName as string,
-    lastName: body.lastName as string,
+  const newGuest = {
+    firstName: body.firstName,
+    lastName: body.lastName,
     attending: false,
   };
 
-  const allGuests = addGuest(guest);
+  const guest = await addGuest(newGuest);
 
-  return Response.json(allGuests);
+  return Response.json(guest);
```

Comment:
this is a bit inconsistent to the other `GET /guests` API route, where we return an object:

```ts
  return Response.json(
    { guests: guests },
```

let's also return an object with the key `guest` here, and the guest inside

Reply:
fixed, and return an object with key `guest` [a2fde60](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/a2fde607a42aa392b94c3612b4662b8c5eb572f5)